### ISSUE TITLE
WCAG A 2.1.2: Unable to pick date and focus getting trapped in calendar when talkback is enabled.

### ISSUE THREAD

**Issue by tanvir2991** [ngx-formly/ngx-formly#3029]
OS: Android

Principle: Operable

Guideline/Success criterion: https://www.w3.org/TR/WCAG21/#no-keyboard-trap

Testing details:

Assistive technology/tool used: Talkback

Browser: Chrome

Test prerequisites: 	

Go to Settings->Convenience tool->Accessibility->Talkback(Enable talkback)

Test steps:

Go to the URL: https://formly.dev/ui/material

Flick your finger right to browse through the screen. Once the control is on the date picker field, double-tap to open the calendar and select a date.

Flick your finger right/left to move out of the calendar.

Issue description: 

Once the user is inside the calendar, control gets trapped there making it impossible for the user to move to other elements when talkback is enabled.

References: 

Please refer to the vide

https://user-images.githubusercontent.com/28960404/138860458-55fc5c26-a4be-418c-8e42-0c52853513f0.mp4

**Comment by aitboudad** [comment]
Tested with the latest version of Angular Material and it seems working https://main.formly.dev/ui/material.
The issue was related to [Angular Material](https://material.angular.io/) so make sure that you're using `>= v12`.
